specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json

```diff
@@ -276,16 +256,62 @@
         }
       }
     },
-    "/providers/Microsoft.AgFoodPlatform/farmBeatsExtensionDefinitions": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AgFoodPlatform/locations/{locations}/operationResults/{operationResultsId}": {
```

Comment:
You shouldn't need to make changes to this API as you should have moved all customers to the more recent 2021-09-01-preview and retired this one 90 days after the availability of that release.

Reply:
We haven't yet deprecated/retired our existing version yet. We will be working on it in sometime.